agents/meinberg_m1000/meinberg_m1000_agent.py

```diff
@@ -0,0 +1,482 @@
+import os
+from os import environ
+
+import time
+import argparse
+import txaio
+
+from autobahn.twisted.util import sleep as dsleep
+from twisted.internet.defer import inlineCallbacks
+
+from socs.snmp import SNMPTwister
+
+# For logging
+txaio.use_twisted()
+
+on_rtd = os.environ.get('READTHEDOCS') == 'True'
+if not on_rtd:
+    from ocs import ocs_agent, site_config
+
+# Mapping of integer OID values to meaningful strings from M1000 manual
+REFCLOCKSTATE = {0: "refclock is not available",
+                 1: "synchronized",
+                 2: "not synchronized"}
+
+NTPCURRENTSTATE = {0: "not available",
+                   1: "not synchronized",
+                   2: "synchronized"}
+
```

Comment:
The meaningful strings above this point and below this point are somewhat differently constructed... I know they all come from the manual but I think it will look better if we adopt the approach of the strings below for the strings above... i.e. make each string a single word ("notAvailable", "notSynchronized"), and drop "refclock is ".

Reply:
Yeah, I just copied each description verbatim from the manual. Updated to using all single word strings.